Add support for Tuya 2 gang relay `_TZ3000_m8f3z8ju`

peper0 opened the pull request:
## Proposed change

## Additional information


## Checklist

- [x] The changes are tested and work correctly
- [ ] `pre-commit` checks pass / the code has been formatted using Black
- [ ] Tests have been added to verify that the new code works

peper0 committed:
Add support for 2 gang relay TS000F from _TZ3000_m8f3z8ju

peper0 committed:
Update ts000f_switch.py

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,95 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                    Basic.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
```
This would resolve the test failure:
```suggestion
                    TuyaZBOnOffAttributeCluster,
```

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,95 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                INPUT_CLUSTERS: [
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
```
And we can probably use it here too:
```suggestion
                    TuyaZBOnOffAttributeCluster,
```

peper0 committed:
Update zhaquirks/tuya/ts000f_switch.py

Co-authored-by: TheJulianJES <TheJulianJES@users.noreply.github.com>

peper0 committed:
Update zhaquirks/tuya/ts000f_switch.py

Co-authored-by: TheJulianJES <TheJulianJES@users.noreply.github.com>

peper0 committed:
Update ts000f_switch.py

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,95 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+    signature = {
+        MODEL: "TS000F",
+        ENDPOINTS: {
+            # SizePrefixedSimpleDescriptor(endpoint=242, profile=41440, device_type=97, device_version=0, input_clusters=[], output_clusters=[33])
+            # input_clusters=["0x0000","0x0004","0x0005","0x0006","0xe000","0xe001"]
+            # output_clusters=["0x000a","0x0019"]>
```
```suggestion
```

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,95 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                    Ota.cluster_id
+                ],
+            },
+            # SizePrefixedSimpleDescriptor(endpoint=242, profile=41440, device_type=97, device_version=0, input_clusters=[], output_clusters=[33])
+            # input_clusters=["0x0004","0x0005","0x0006","0xe001"]
+            # output_clusters=[]>
```
```suggestion
```

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,95 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                ],
+                OUTPUT_CLUSTERS: [],
+            },
+            # <SimpleDescriptor endpoint=242 profile=41440 device_type=97
+            # input_clusters=[]
+            # output_clusters=["0x0021"]>
```
```suggestion
```

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,95 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                    Ota.cluster_id
+                ],
+            },
+            # SizePrefixedSimpleDescriptor(endpoint=242, profile=41440, device_type=97, device_version=0, input_clusters=[], output_clusters=[33])
+            # input_clusters=["0x0004","0x0005","0x0006","0xe001"]
+            # output_clusters=["0x000a","0x0019"]>
```
```suggestion
```

TheJulianJES committed:
Remove cluster comments

Most of these had endpoint=242, so seem to be incorrect. They're not important anyway, so removing them.

TheJulianJES reviewed (approved):
I think this should be fine. Thanks!

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,83 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    TuyaZBOnOffAttributeCluster,
+                    TuyaZBExternalSwitchTypeCluster
```
```suggestion
                    TuyaZBExternalSwitchTypeCluster,
```

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,83 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                    Scenes.cluster_id,
+                    TuyaZBOnOffAttributeCluster,
+                    TuyaZBE000Cluster,
+                    TuyaZBExternalSwitchTypeCluster
```
```suggestion
                    TuyaZBExternalSwitchTypeCluster,
```

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,83 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                ],
+                OUTPUT_CLUSTERS: [
+                    Time.cluster_id,
+                    Ota.cluster_id
```
```suggestion
                    Ota.cluster_id,
```

TheJulianJES committed:
Apply changes for pre-commit

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,83 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    TuyaZBExternalSwitchTypeCluster.cluster_id
```
```suggestion
                    TuyaZBExternalSwitchTypeCluster.cluster_id,
```

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,83 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                    Scenes.cluster_id,
+                    OnOff.cluster_id,
+                    TuyaZBE000Cluster.cluster_id,
+                    TuyaZBExternalSwitchTypeCluster.cluster_id
```
```suggestion
                    TuyaZBExternalSwitchTypeCluster.cluster_id,
```

TheJulianJES commented on zhaquirks/tuya/ts000f_switch.py:
```diff
@@ -97,3 +98,83 @@ class Tuya_1G_Wall_Switch_Metering(EnchantedDevice):
+                ],
+                OUTPUT_CLUSTERS: [
+                    Time.cluster_id,
+                    Ota.cluster_id
```
```suggestion
                    Ota.cluster_id,
```

TheJulianJES committed:
Apply changes for pre-commit
